homeassistant/components/camera/prefs.py

```diff
@@ -71,16 +50,22 @@ async def async_update(
     ) -> dict[str, bool | Orientation]:
```

Comment:
Not sure why stream_options is here. Seems like we can just remove it?

Reply:
Agree, i just had a quick look at the code and it seems unused and removing looks safe.